src/transaction.rs

```diff
@@ -420,7 +416,7 @@ impl Transaction {
                     ..
                 }
                 | Input::MessageSigned {
-                    owner,
+                    recipient: owner,
```

Comment:
Since there's already a separate method to fetch the recipient, it is confusing to return a field for messages here which shouldn't exist. `.owner()` should return None for messages.

Reply:
doing this now breaks two tests, since they return None when `sign_inputs` with `Input::MessageSigned` we end up with empty `witness` data